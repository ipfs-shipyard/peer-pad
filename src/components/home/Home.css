@import '../../colors.css';

.caribbean-green-glow {
  text-shadow:
    var(--caribbean-green) 0 0 60px,
    var(--caribbean-green) 0 0 40px,
    var(--caribbean-green) 0 0 20px,
    var(--caribbean-green) 0 0 10px;
  position: relative;
  display: inline-block;
}

.caribbean-green-glow::after {
  content: '';
  position: absolute;
  left: 50%;
  top: 50%;
  display: block;
  width: 0px;
  height: 0px;
  box-shadow: var(--caribbean-green) 0 0 500px 30px;
  border-radius: 100%;
  z-index: -900;
}

.bright-turquoise-glow {
  text-shadow:
    var(--bright-turquoise) 0 0 60px,
    var(--bright-turquoise) 0 0 40px,
    var(--bright-turquoise) 0 0 20px,
    var(--bright-turquoise) 0 0 10px;
  position: relative;
  display: inline-block;
}

.bright-turquoise-glow::after {
  content: '';
  position: absolute;
  left: 50%;
  top: 50%;
  display: block;
  width: 0px;
  height: 0px;
  box-shadow: var(--bright-turquoise) 0 0 500px 30px;
  border-radius: 100%;
  z-index: -900;
}

.bigger-when-small {
  width: 200%;
  margin: 0 auto;
  position: relative;
  left: -50%;
  max-width: initial;
}

.hover--underline-thick {
  position: relative;
}

.hover--underline-thick:after {
  content: '';
  width: 100%;
  height: 3px;
  position: absolute;
  left: 0;
  bottom: 0;
  opacity: 0;
  background-color: var(--caribbean-green);
  transition: opacity .15s, transform .15s;
  transform: translateY(8px);
}

.hover--underline-thick:hover:after {
  opacity: 1;
  transform: translateY(0);
}

.Home a {
  color: #fff;
  text-decoration: none;
  border-bottom: 1px solid rgba(2, 202, 173, 0.2)
}

.Home a:hover {
  border-bottom: 1px solid var(--caribbean-green)
}

.Home header a,
.Home header a:hover {
  border-bottom: none
}

.Home footer nav a {
  border-bottom-color: transparent
}

@media screen and (min-width: 680px) {
  /* The big wave backgrounds are heavy and complex, so skip them on mobile devices. */
  .bg-hero-wave {
    position: relative;
    z-index: 200;
    height: auto;
  }

  .bg-hero-wave-inner {
    content: '';
    display: block;
    z-index: -100;
    position: absolute;
    top: 0;
    left: 0%;
    width: 100%;
    height: 100%;
    background: transparent center no-repeat;
    background-size: 100% auto;
    opacity: 0.2;
    mix-blend-mode: soft-light;
  }

 .bigger-when-small {
    width: 100%;
    margin: 0 auto;
    position: static;
    left: 0%;
    max-width: 100%;
 }
}

@media screen and (min-width: 1024px) {
  /* The spot lights make the text hard to read on small screens so we dial them
   * down by default and dial them up on larger screens.
  */
  .bright-turquoise-glow::after {
    width: 400px;
    box-shadow: var(--bright-turquoise) -350px -20px 1000px 22px;
  }
 .caribbean-green-glow::after {
    width: 400px;
    box-shadow: var(--caribbean-green) -250px -20px 1000px 22px;
  }
}
