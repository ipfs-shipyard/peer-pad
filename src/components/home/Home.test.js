/* global it, expect */
import React from 'react'
import { mount } from 'enzyme'
import { createMemoryHistory } from 'history'
import { Router } from 'react-router'
import toJSON from 'enzyme-to-json'
import Home from './Home'

it('can create document via button', (done) => {
  const history = createMemoryHistory()
  const all = mount(
    <Router history={history}><Home /></Router>)

  const home = all.find(Home)
  expect(toJSON(home)).toMatchSnapshot()

  const unlisten = history.listen((location, action) => {
    expect(location.pathname).toMatch(/^\/w\/markdown\/\w+\/\w+\-\w+$/)
    unlisten()
    done()
  })

  const button = home.find('button').last()
  button.simulate('click')
})
