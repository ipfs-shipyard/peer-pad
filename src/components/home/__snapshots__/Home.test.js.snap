// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can create document via button 1`] = `
<Home>
  <div
    className="Home tc white"
    style={
      Object {
        "backgroundColor": "#041125",
      }
    }
  >
    <Component>
      <div
        className="bg-razzmatazz white"
        style={
          Object {
            "padding": "20px",
          }
        }
      >
        <Component
          className="dib fill--current-color v-mid mr3"
        >
          <svg
            className="dib fill--current-color v-mid mr3"
            height="26"
            viewBox="0 0 149 146.7"
            width="26"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M119.38 117.8a30 30 0 0 0 3.8-8.39 47.86 47.86 0 0 0 1.48-8.37c.14-1.49.28-3.24.3-4.95a.53.53 0 0 0-.56-.58c-1.58-.07-3.19-.06-4.79 0a.58.58 0 0 0-.53.58c0 .44-.08.88-.1 1.32s0 .73-.08 1.09a22.82 22.82 0 0 1-1.06 5.28 6.56 6.56 0 0 1-4.95 4.35 6.88 6.88 0 0 1-6.43-1.92 18.9 18.9 0 0 1-3.05-4.64 53.77 53.77 0 0 1-3.47-10.65 3.36 3.36 0 0 1 .06-1.68c2.25-8.21 4.11-16.28 5.64-23.1s3.09-13.74 4.64-20.61q2.15-9.51 4.28-19a1.42 1.42 0 0 0 0-.58c-.06-.24-.13-.49-.85-.49H96.93c-.77 0-.93.13-1.1.89l-4 18.16-2 5.63L87 44c-.25-.56-.48-1.1-.72-1.63a51 51 0 0 0-6.91-11.5 22.5 22.5 0 0 0-8.7-6.53 24.46 24.46 0 0 0-9.57-1.94 27.43 27.43 0 0 0-4.79.43 34.58 34.58 0 0 0-17.76 9 41.61 41.61 0 0 0-11.17 18.25A72.85 72.85 0 0 0 24.06 71a89.74 89.74 0 0 0 1.27 17.82 52.69 52.69 0 0 0 6.5 18.29 35.92 35.92 0 0 0 8.07 9.41 34.41 34.41 0 0 0 17.93 7.54 36.91 36.91 0 0 0 12.74-.91 31.43 31.43 0 0 0 9.6-4 35.18 35.18 0 0 0 9-8.14c.49-.62 1-1.24 1.44-1.89l3.15-4.12 1.56 3.81-2.17.89 2.23-.77.14.35a39.84 39.84 0 0 0 3.53 7.72 17.35 17.35 0 0 0 5.18 5.7 9.87 9.87 0 0 0 11.25-.44 17.54 17.54 0 0 0 3.9-4.46zm-34.3-41.58c-.57 2.12-1.09 4.26-1.6 6.4s-1 4-1.49 6a66.81 66.81 0 0 1-4.55 12.72 28.46 28.46 0 0 1-6.13 8.66 13.36 13.36 0 0 1-9.25 3.78h-.68c-4.86 0-8.76-2.26-11.61-6.73a31.35 31.35 0 0 1-3.9-9.76 73.72 73.72 0 0 1-1.79-12 144.6 144.6 0 0 1-.28-16 96.75 96.75 0 0 1 2-17.6A37.79 37.79 0 0 1 49.1 42a17.82 17.82 0 0 1 5.44-6.58 13.25 13.25 0 0 1 8-2.49 7.51 7.51 0 0 1 4.4 1.7 18.54 18.54 0 0 1 3.81 3.92 61.7 61.7 0 0 1 6.76 11.95 205.86 205.86 0 0 1 7.44 20 9.59 9.59 0 0 1 .13 5.72z"
            />
          </svg>
        </Component>
        <span
          className="dib-ns v-mid f6 fw6 "
        >
          Peerpad is in Alpha. 
        </span>
        <span
          className="db dib-ns pt3 pt0-l v-mid f6 fw3"
        >
          The codebase hasn't been audited by Security specialists and it shouldn't be used to store, share or publish sensitive information.
        </span>
      </div>
    </Component>
    <header
      className="db pv3 center"
      style={
        Object {
          "maxWidth": "1260px",
        }
      }
    >
      <nav
        className="db dt-l w-100 border-box pa3 ph5-l"
      >
        <a
          className="dim db dtc-l v-btm mid-gray link w-100 w-25-l tc tl-l mb2 mb0-l"
          href="/"
          title="Home"
        >
          <img
            alt="PeerPad"
            className="dib"
            src="images/logo-peerpad-lg.svg"
            style={
              Object {
                "height": "100px",
              }
            }
          />
        </a>
        <div
          className="db dtc-l v-btm w-100 w-75-l tc tr-l"
        >
          <HashLink
            className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
            title="About"
            to="#about"
          >
            <Link
              className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
              onClick={[Function]}
              replace={false}
              title="About"
              to="#about"
            >
              <a
                className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
                href="#about"
                onClick={[Function]}
                title="About"
              >
                What is PeerPad
              </a>
            </Link>
          </HashLink>
          <HashLink
            className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
            title="Features"
            to="#features"
          >
            <Link
              className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
              onClick={[Function]}
              replace={false}
              title="Features"
              to="#features"
            >
              <a
                className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
                href="#features"
                onClick={[Function]}
                title="Features"
              >
                Features
              </a>
            </Link>
          </HashLink>
          <HashLink
            className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
            title="How it works"
            to="#how-it-works"
          >
            <Link
              className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
              onClick={[Function]}
              replace={false}
              title="How it works"
              to="#how-it-works"
            >
              <a
                className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
                href="#how-it-works"
                onClick={[Function]}
                title="How it works"
              >
                How it works
              </a>
            </Link>
          </HashLink>
          <HashLink
            className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
            title=""
            to="#benefits"
          >
            <Link
              className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
              onClick={[Function]}
              replace={false}
              title=""
              to="#benefits"
            >
              <a
                className="hover--underline-thick white f6 f5-l fw3 db dib-ns mr4-ns pt3 pb3"
                href="#benefits"
                onClick={[Function]}
                title=""
              >
                Benefits
              </a>
            </Link>
          </HashLink>
          <a
            className="hover--underline-thick white f6 fw3 db dib-ns mr4-ns pt3 pb3"
            href="https://github.com/ipfs-shipyard/peerpad"
            title="Open Source"
          >
            Open Source
          </a>
          <div
            className="db dib-ns pv3 pv0-ns"
          >
            <Component>
              <CreateDocumentContainer>
                <StartButton
                  onClick={[Function]}
                >
                  <button
                    className="dib bg-transparent ba b--bright-turquoise bright-turquoise fw2 tracked--1 pointer dim"
                    data-id="start-button"
                    onClick={[Function]}
                    style={
                      Object {
                        "fontSize": "15px",
                        "padding": "5px 28px",
                      }
                    }
                    type="button"
                  >
                    START
                  </button>
                </StartButton>
              </CreateDocumentContainer>
            </Component>
          </div>
        </div>
      </nav>
    </header>
    <section
      className="db pt4 pb6 center bg-hero-wave"
      id="hero"
      style={
        Object {
          "maxWidth": "1440px",
        }
      }
    >
      <div
        className="bg-hero-wave-inner"
        style={
          Object {
            "backgroundImage": "url(images/hero-wave.png)",
          }
        }
      />
      <div
        className="center ph3"
        style={
          Object {
            "maxWidth": "60rem",
          }
        }
      >
        <h1
          className="hero-cursor f2 f1-ns fw3 lh-copy tracked--1 bright-turquoise"
        >
          HELLO WORLD!
        </h1>
        <h2
          className="f3 f2-ns fw2 lh-copy tracked--2 relative"
          style={
            Object {
              "zIndex": "1000",
            }
          }
        >
          PeerPad is a realtime P2P collaborative 
          <span
            className="hero-editing"
          >
            editing
          </span>
           tool, powered by 
          <span
            className="fw4 bright-turquoise-glow"
          >
            <a
              href="//ipfs.io"
              style={
                Object {
                  "borderBottom": 0,
                  "textDecoration": "none",
                }
              }
            >
              IPFS
            </a>
          </span>
           and 
          <span
            className="fw4 caribbean-green-glow"
          >
            <a
              href="http://y-js.org/"
              style={
                Object {
                  "borderBottom": 0,
                  "textDecoration": "none",
                }
              }
            >
              CRDTs
            </a>
          </span>
        </h2>
        <div
          className="mt6"
        >
          <Component>
            <CreateDocumentContainer>
              <CreateDocument
                onCreateDocument={[Function]}
                onTypeChange={[Function]}
                type="markdown"
              >
                <form>
                  <div
                    className="lh-copy"
                  >
                    <label
                      className="db dib-ns fw2 tracked white mv2"
                      style={
                        Object {
                          "fontSize": "28px",
                        }
                      }
                    >
                      Create new:
                    </label>
                    <select
                      className="appearance-none fw1 tracked--1 mh3 mv2 ph3 pv2 ba b--dashed b--caribbean-green-soft white tc"
                      onChange={[Function]}
                      placeholder="select type"
                      style={
                        Object {
                          "background": "none",
                          "borderRadius": 0,
                          "fontSize": "18px",
                        }
                      }
                      value="markdown"
                    >
                      <DocTypeOption
                        text="Markdown pad"
                        value="markdown"
                      >
                        <option
                          className="black"
                          style={
                            Object {
                              "fontSize": "18px",
                            }
                          }
                          value="markdown"
                        >
                          Markdown pad
                        </option>
                      </DocTypeOption>
                      <DocTypeOption
                        text="Rich text pad"
                        value="richtext"
                      >
                        <option
                          className="black"
                          style={
                            Object {
                              "fontSize": "18px",
                            }
                          }
                          value="richtext"
                        >
                          Rich text pad
                        </option>
                      </DocTypeOption>
                      <DocTypeOption
                        text="Math pad"
                        value="math"
                      >
                        <option
                          className="black"
                          style={
                            Object {
                              "fontSize": "18px",
                            }
                          }
                          value="math"
                        >
                          Math pad
                        </option>
                      </DocTypeOption>
                    </select>
                    <button
                      className="input-reset bg-caribbean-green ba bw2 b--caribbean-green white f6 fw5 mv2 ph3 pv2 pointer dim tracked--1"
                      onClick={[Function]}
                      type="button"
                    >
                      GO
                    </button>
                  </div>
                </form>
              </CreateDocument>
            </CreateDocumentContainer>
          </Component>
        </div>
      </div>
    </section>
    <section
      className="pa3 mv4 mv5-ns mw8 center"
      id="about"
    >
      <h2
        className="f1-ns fw2 lh-copy tracked--1 fancy-underline"
        style={
          Object {
            "fontSize": "40px",
          }
        }
      >
        What is PeerPad?
      </h2>
      <p
        className="f4 f3-ns fw1 tl tc-ns lh-copy tracked--1 center"
      >
        PeerPad is a collaborative real-time editor that works on the decentralised web, built on top of 
        <a
          href="//ipfs.io"
        >
          IPFS
        </a>
         and 
        <a
          href="http://y-js.org/"
        >
          Y.js
        </a>
        . It uses no second or third-party: all participating nodes talk directly to each other without a central service. Peerpad is 
        <a
          href="//github.com/ipfs-shipyard/peerpad"
        >
          open-source
        </a>
         and built by 
        <a
          href="//protocol.ai"
        >
          Protocol Labs
        </a>
         and the 
        <a
          href="//ipfs.io"
        >
          IPFS community
        </a>
        .
      </p>
    </section>
    <section
      className="pa3 mv4 mv5-ns"
      id="features"
    >
      <div
        className="mw8 center"
      >
        <h2
          className="f1-ns fw2 lh-copy tracked--1 fancy-underline"
          style={
            Object {
              "fontSize": "40px",
            }
          }
        >
          Features
        </h2>
        <p
          className="f4 f3-ns fw1 tl tc-ns lh-copy tracked--1 center"
        >
          PeerPad does not rely on a second or third-party.
        </p>
        <p
          className="f4 f3-ns fw1 tl tc-ns lh-copy tracked--1 center"
        >
          All nodes talk to each other directly, without intermediation.
        </p>
      </div>
      <div
        className="cf center tc"
        id="features-list"
        style={
          Object {
            "maxWidth": "1200px",
          }
        }
      >
        <div
          className="feature-item fl-ns w-50-m w-25-l pa4 o-20"
          title="Peerpad is in Alpha and has not been audited by security experts yet."
        >
          <Hexicon
            name="Private"
          >
            <div
              className="hexicon dt caribbean-green center relative"
              style={
                Object {
                  "background": "transparent url(images/hex.svg) center no-repeat",
                  "height": 124,
                  "width": 108,
                }
              }
            >
              <div
                className="dtc v-mid tc w-100"
              >
                <img
                  alt="Private"
                  className="center v-mid"
                  src="images/private.svg"
                  style={
                    Object {
                      "maxWidth": 60,
                    }
                  }
                />
              </div>
            </div>
          </Hexicon>
          <h3
            className="f4 fw1 white tracked--1"
          >
            Private
          </h3>
          <p
            className="f6 fw2 lh-copy ph2"
          >
            Communication between parties is encrypted.
          </p>
        </div>
        <div
          className="feature-item fr-ns w-50-m w-25-l pa4"
        >
          <Hexicon
            name="Encrypted"
          >
            <div
              className="hexicon dt caribbean-green center relative"
              style={
                Object {
                  "background": "transparent url(images/hex.svg) center no-repeat",
                  "height": 124,
                  "width": 108,
                }
              }
            >
              <div
                className="dtc v-mid tc w-100"
              >
                <img
                  alt="Encrypted"
                  className="center v-mid"
                  src="images/encrypted.svg"
                  style={
                    Object {
                      "maxWidth": 60,
                    }
                  }
                />
              </div>
            </div>
          </Hexicon>
          <h3
            className="f4 fw1 white tracked--1"
          >
            Encrypted
          </h3>
          <p
            className="f6 fw2 lh-copy o-70 ph2"
          >
            Access to content depends on a secret key. A node needs access to a “read” key to read a doc and it can only change content with access to a “write” key.
          </p>
        </div>
        <div
          className="feature-item fl-ns w-50-m w-25-l pa4"
          style={
            Object {
              "clear": "both",
            }
          }
        >
          <Hexicon
            name="Collaborative"
          >
            <div
              className="hexicon dt caribbean-green center relative"
              style={
                Object {
                  "background": "transparent url(images/hex.svg) center no-repeat",
                  "height": 124,
                  "width": 108,
                }
              }
            >
              <div
                className="dtc v-mid tc w-100"
              >
                <img
                  alt="Collaborative"
                  className="center v-mid"
                  src="images/collaborative.svg"
                  style={
                    Object {
                      "maxWidth": 60,
                    }
                  }
                />
              </div>
            </div>
          </Hexicon>
          <h3
            className="f4 fw1 white tracked--1"
          >
            Collaborative
          </h3>
          <p
            className="f6 fw2 lh-copy o-70 ph2"
          >
            Thanks to 
            <a
              href="https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type"
            >
              CRDTs
            </a>
             and 
            <a
              href="http://y-js.org/"
            >
              Y.js
            </a>
            , several authors can collaborate and edit a document without conflicts, even when they aren’t connected at all times.
          </p>
        </div>
        <div
          className="feature-item fr-ns w-50-m w-25-l pa4"
        >
          <Hexicon
            name="Realtime"
          >
            <div
              className="hexicon dt caribbean-green center relative"
              style={
                Object {
                  "background": "transparent url(images/hex.svg) center no-repeat",
                  "height": 124,
                  "width": 108,
                }
              }
            >
              <div
                className="dtc v-mid tc w-100"
              >
                <img
                  alt="Realtime"
                  className="center v-mid"
                  src="images/realtime.svg"
                  style={
                    Object {
                      "maxWidth": 60,
                    }
                  }
                />
              </div>
            </div>
          </Hexicon>
          <h3
            className="f4 fw1 white tracked--1"
          >
            Realtime
          </h3>
          <p
            className="f6 fw2 lh-copy o-70 ph2"
          >
            When multiple people edit a document while connected to each other, everyone’s changes are reflected in the document in real-time.
          </p>
        </div>
      </div>
    </section>
    <section
      className="pa3 mv4 mv5-ns mw8 center"
      id="how-it-works"
    >
      <h2
        className="f1-ns fw2 lh-copy tracked--1 fancy-underline"
        style={
          Object {
            "fontSize": "40px",
          }
        }
      >
        How it works
      </h2>
      <p
        className="f4 f3-ns fw1 tl tc-ns lh-copy tracked--1 center"
      >
        A Conflict-free Replicated Data Type (CRDT) offers 'Strong Eventual Consistency': a flavour of eventual consistency that ensures conflicts can be merged automatically to produce a value that is guaranteed to be correct/consistent.
      </p>
      <p
        className="f4 f3-ns fw1 tl tc-ns lh-copy tracked--1 center"
      >
        Two objects can be either equal, have hierarchy (one descends the other) or are pairs; the latter signifies a branch/divergence/conflict. From the intrinsic state of the two pairs, we can determine a new descendant object which is the result of the merge.
      </p>
      <div
        style={
          Object {
            "overflow": "hidden",
          }
        }
      >
        <img
          alt="A graph showing peers independently updating state over time; The most recent state is the union of each peers updates."
          className="bigger-when-small"
          src="images/how-it-works.png"
        />
      </div>
    </section>
    <section
      className="pa3 mv4 mv5-ns"
      id="benefits"
    >
      <div
        className="mw8 center mb5 mb6-ns"
      >
        <h2
          className="f1-ns fw2 lh-copy tracked--1 fancy-underline"
          style={
            Object {
              "fontSize": "40px",
            }
          }
        >
          What you can do with PeerPad
        </h2>
        <p
          className="f4 f3-ns fw1 tl tc-ns lh-copy tracked--1 center"
        >
          Peerpad can be used to edit code, markdown documents or even rich text documents. Peerpad can also be used to save snapshots and publish them to the internet.
        </p>
      </div>
      <div
        className="center tl mw8"
      >
        <div
          className="dib-l w-50-l v-top pb4 mw6 center"
        >
          <div
            className="dt w-100"
          >
            <div
              className="dtc v-top"
              style={
                Object {
                  "width": 108,
                }
              }
            >
              <Hexicon
                name="Take notes"
              >
                <div
                  className="hexicon dt caribbean-green center relative"
                  style={
                    Object {
                      "background": "transparent url(images/hex.svg) center no-repeat",
                      "height": 124,
                      "width": 108,
                    }
                  }
                >
                  <div
                    className="dtc v-mid tc w-100"
                  >
                    <img
                      alt="Take notes"
                      className="center v-mid"
                      src="images/take-notes.svg"
                      style={
                        Object {
                          "maxWidth": 60,
                        }
                      }
                    />
                  </div>
                </div>
              </Hexicon>
            </div>
            <div
              className="dtc v-top pl3 pr4-l"
            >
              <h3
                className="f4 fw1 white tracked--1"
              >
                Take meeting notes
              </h3>
              <p
                className="f6 fw2 lh-copy o-70"
              >
                Either using plain text, Markdown or Rich-text, you can take meeting notes and share with your colleagues in real time..
              </p>
            </div>
          </div>
        </div>
        <div
          className="dib-l w-50-l v-top pb4 mw6 center"
        >
          <div
            className="dt w-100"
          >
            <div
              className="dtc v-top"
              style={
                Object {
                  "width": 108,
                }
              }
            >
              <Hexicon
                name="Collaborate"
              >
                <div
                  className="hexicon dt caribbean-green center relative"
                  style={
                    Object {
                      "background": "transparent url(images/hex.svg) center no-repeat",
                      "height": 124,
                      "width": 108,
                    }
                  }
                >
                  <div
                    className="dtc v-mid tc w-100"
                  >
                    <img
                      alt="Collaborate"
                      className="center v-mid"
                      src="images/collaborate.svg"
                      style={
                        Object {
                          "maxWidth": 60,
                        }
                      }
                    />
                  </div>
                </div>
              </Hexicon>
            </div>
            <div
              className="dtc v-top pl3 pr4-l"
            >
              <h3
                className="f4 fw1 white tracked--1"
              >
                Collaborate or share code snippets
              </h3>
              <p
                className="f6 fw2 lh-copy o-70"
              >
                Peerpad has a built-in code editor you can use to collaborate with colleagues while editing the same file.
              </p>
            </div>
          </div>
        </div>
        <div
          className="dib-l w-50-l v-top pb4 mw6 center"
        >
          <div
            className="dt w-100"
          >
            <div
              className="dtc v-top"
              style={
                Object {
                  "width": 108,
                }
              }
            >
              <Hexicon
                name="Write articles"
              >
                <div
                  className="hexicon dt caribbean-green center relative"
                  style={
                    Object {
                      "background": "transparent url(images/hex.svg) center no-repeat",
                      "height": 124,
                      "width": 108,
                    }
                  }
                >
                  <div
                    className="dtc v-mid tc w-100"
                  >
                    <img
                      alt="Write articles"
                      className="center v-mid"
                      src="images/write-articles.svg"
                      style={
                        Object {
                          "maxWidth": 60,
                        }
                      }
                    />
                  </div>
                </div>
              </Hexicon>
            </div>
            <div
              className="dtc v-top pl3 pr4-l"
            >
              <h3
                className="f4 fw1 white tracked--1"
              >
                Write articles and share them
              </h3>
              <p
                className="f6 fw2 lh-copy o-70"
              >
                You can publish a snapshot of a pad to 
                <a
                  href="//ipfs.io"
                >
                  IPFS
                </a>
                , making it available on the internet. Choose who you share them with by sharing a read key that decrypts the content.
              </p>
            </div>
          </div>
        </div>
        <div
          className="dib-l w-50-l v-top pb4 mw6 center"
        >
          <div
            className="dt w-100"
          >
            <div
              className="dtc v-top"
              style={
                Object {
                  "width": 108,
                }
              }
            >
              <Hexicon
                name="Work with multiple users"
              >
                <div
                  className="hexicon dt caribbean-green center relative"
                  style={
                    Object {
                      "background": "transparent url(images/hex.svg) center no-repeat",
                      "height": 124,
                      "width": 108,
                    }
                  }
                >
                  <div
                    className="dtc v-mid tc w-100"
                  >
                    <img
                      alt="Work with multiple users"
                      className="center v-mid"
                      src="images/work-with-multiple-users.svg"
                      style={
                        Object {
                          "maxWidth": 60,
                        }
                      }
                    />
                  </div>
                </div>
              </Hexicon>
            </div>
            <div
              className="dtc v-top pl3 pr4-l"
            >
              <h3
                className="f4 fw1 white tracked--1"
              >
                Work with multiple users
              </h3>
              <p
                className="f6 fw2 lh-copy o-70"
              >
                Peerpad can work with many users changing the document at the same time, seing each other’s changes in real-time.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
    <footer
      className="ph4 pv4 pv6-l"
      style={
        Object {
          "background": "transparent url(images/footer.png) center bottom no-repeat",
        }
      }
    >
      <div
        className="cf w-100 center pv3 bt bw1 b--caribbean-green-soft"
        style={
          Object {
            "borderTopStyle": "dashed",
            "maxWidth": "1140px",
          }
        }
      >
        <nav
          className="db fl w-100 w-50-l tl-l"
        >
          <div>
            <HashLink
              className="hover--underline-thick white f6 fw3 dib mr4 m3t pb2"
              title="About"
              to="#about"
            >
              <Link
                className="hover--underline-thick white f6 fw3 dib mr4 m3t pb2"
                onClick={[Function]}
                replace={false}
                title="About"
                to="#about"
              >
                <a
                  className="hover--underline-thick white f6 fw3 dib mr4 m3t pb2"
                  href="#about"
                  onClick={[Function]}
                  title="About"
                >
                  About
                </a>
              </Link>
            </HashLink>
            <HashLink
              className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
              title="Features"
              to="#features"
            >
              <Link
                className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
                onClick={[Function]}
                replace={false}
                title="Features"
                to="#features"
              >
                <a
                  className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
                  href="#features"
                  onClick={[Function]}
                  title="Features"
                >
                  Features
                </a>
              </Link>
            </HashLink>
            <HashLink
              className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
              title="How it works"
              to="#how-it-works"
            >
              <Link
                className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
                onClick={[Function]}
                replace={false}
                title="How it works"
                to="#how-it-works"
              >
                <a
                  className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
                  href="#how-it-works"
                  onClick={[Function]}
                  title="How it works"
                >
                  How it works
                </a>
              </Link>
            </HashLink>
            <HashLink
              className="hover--underline-thick white f6 fw3 dib mt3 pb2"
              title="What you can do with PeerPad"
              to="#benefits"
            >
              <Link
                className="hover--underline-thick white f6 fw3 dib mt3 pb2"
                onClick={[Function]}
                replace={false}
                title="What you can do with PeerPad"
                to="#benefits"
              >
                <a
                  className="hover--underline-thick white f6 fw3 dib mt3 pb2"
                  href="#benefits"
                  onClick={[Function]}
                  title="What you can do with PeerPad"
                >
                  Benefits
                </a>
              </Link>
            </HashLink>
          </div>
          <div>
            <a
              className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
              href="https://github.com/ipfs-shipyard/peerpad/blob/master/docs/ARCHITECTURE.md"
              title="Architecture"
            >
              Architecture
            </a>
            <a
              className="hover--underline-thick white f6 fw3 dib mr4 mt3 pb2"
              href="https://github.com/ipfs-shipyard/peerpad/blob/master/docs/SECURITY.md"
              title="Security"
            >
              Security
            </a>
            <a
              className="hover--underline-thick white f6 fw3 dib mt3 pb2"
              href="https://github.com/ipfs-shipyard/peerpad"
              title="Open Source"
            >
              Open Source
            </a>
          </div>
        </nav>
        <small
          className="fl db w-100 w-50-l tr-l f7 f6-ns pv4 pv0-l lh-copy mt3"
        >
          <span
            className="db dib-ns mb3 mb0-ns center"
          >
            <a
              className="link white"
              href="https://protocol.ai/"
            >
              © Protocol Labs 
            </a>
          </span>
          <span
            className="dn dib-ns mh1"
          >
             | 
          </span>
          Except as 
          <a
            className="link bright-turquoise"
            href="https://protocol.ai/legal/"
          >
            noted
          </a>
          , content licensed 
          <a
            className="link bright-turquoise"
            href="https://creativecommons.org/licenses/by/3.0/"
          >
            CC-BY 3.0
          </a>
        </small>
      </div>
    </footer>
  </div>
</Home>
`;
